example/ssd/dataset/pycocotools/coco.py

```diff
@@ -255,22 +255,23 @@ def showAnns(self, anns):
                             color.append(c)
                     else:
                         # mask
-                        t = self.imgs[ann['image_id']]
-                        if type(ann['segmentation']['counts']) == list:
+                        raise NotImplementedError("maskUtils disabled!")
+                        # t = self.imgs[ann['image_id']]
+                        # if type(ann['segmentation']['counts']) == list:
                             # rle = maskUtils.frPyObjects([ann['segmentation']], t['height'], t['width'])
-                            raise NotImplementedError("maskUtils disabled!")
-                        else:
-                            rle = [ann['segmentation']]
+                        #    raise NotImplementedError("maskUtils disabled!")
```

Comment:
If these are not needed, I would prefer just removing them instead of commenting out.